config/routes.rb

```diff
@@ -20,7 +20,7 @@ def add_swagger_route http_method, path, opts = {}
       namespace :v0x0, :path => "v0.0" do
         full_path = File.join(ENV["BASE_PATH"], full_path) if ENV["BASE_PATH"]
         constraint = opts[:constraint_name].camelize.constantize
-        match full_path, to: "#{opts.fetch(:controller_name)}##{opts[:action_name]}", constraints: constraint, via: http_method
```

Comment:
Why are we pulling `constraints` out?

Reply:
@syncrou From our conversation this was a temporary fix to make sure we get the 403 and 401s working while we wait for a final design. If I don't remove the constraint it fails with the ambiguous 404

Reply:
The constraint checks that the header matches:
```
 { 'x-rh-auth-identity': { 'identity' => { 'is_org_admin':false } } } 
```
There is no admin to authenticate. If there was an account id in the header that would be parsed by our future authentication library down the chain from the constraint. 
The authentication library would handle returning a 401, 403.

If a request comes through without the above header than a 404 is the appropriate response.

Reply:
reverted the constraint changed